stable/cert-manager/templates/rbac.yaml

```diff
@@ -13,12 +13,7 @@ rules:
     resources: ["certificates", "issuers", "clusterissuers"]
     verbs: ["*"]
   - apiGroups: [""]
-    # TODO: remove endpoints once 0.4 is released. We include it here in case
-    # users use the 'master' version of the Helm chart with a 0.2.x release of
-    # cert-manager that still performs leader election with Endpoint resources.
-    # We advise users don't do this, but some will anyway and this will reduce
-    # friction.
-    resources: ["endpoints", "configmaps", "secrets", "events", "services", "pods"]
+    resources: ["configmaps", "secrets", "events", "services", "pods"]
```

Comment:
This means we drop support for cert-manager v0.2.0-v0.2.2 (iirc). In general, our rule is v0.4.x of the chart is compatible with v0.4.y of cert-manager (i.e. we do not pin the patch version of appVersion and version, but we *do* pin minor versions).